README.md

```diff
@@ -159,15 +159,34 @@ any errors encountered during the request.
 
 ## Body
 
-If the server's response contains a body it should follow the requirements for [GraphQL response](https://graphql.github.io/graphql-spec/June2018/#sec-Response).
+If the server's response contains a body it MUST follow the requirements for [GraphQL response](https://graphql.github.io/graphql-spec/June2018/#sec-Response).
 
-Note: For any non-2XX response, the client should not rely on the body to be in GraphQL format since the source of the response
-may not be the GraphQL server but instead some intermediary such as API gateways, firewalls, etc.
+Note: For any non-`200` response, the client should not rely on the body to be in GraphQL format.
+The source of the response may not be the GraphQL server, but instead some intermediary such as API gateways, firewalls, etc.
 
 ## Status Codes
 
-If the `data` entry in the response has any value other than `null` (when the operation has successfully executed
-without error) then the response should use the 200 (OK) status code.
-If the operation failed before or during execution, due to a syntax error, missing information, validation error
-or any other reason, then response should use the 4XX or 5XX status codes.
-It is recommended to use the same error codes as the [reference implementation](https://github.com/graphql/express-graphql).
+If the server successfully executed parts of the operation and returns a well-formed response
+that contains a non-null key `data`, the status code MUST be `200`.
+
+If the status code is `200`, the response MUST be a well-formed GraphQL response.
+
+The result of executing a GraphQL operation may contain partial data as well as encountered errors.
+Errors that happen during execution of the GraphQL operation typically become part of the result,
+as long as the server is still able to produce a well-formed response.
+
+In case of errors that completely prevent the successful execution of the request, the server SHOULD respond
```

Comment:
As discussed in today's working group meeting, it is my belief this should be a `MUST`.
```suggestion
In case of errors that completely prevent the successful execution of the request, the server MUST respond
```

We should change the following phrasing from using the word "example", to concretely outlining the situations the status is `400`, or `200` — as outlined by @benjie here: https://github.com/APIs-guru/graphql-over-http/pull/79/files#r423698052 .